### PR TITLE
Create deployment_instruction.txt

### PR DESCRIPTION
**Pull request by atiabjobayer**
Changes: Video instructions for deploying chat.susi.ai on surge server

### CODE REVIEW

File: docs/deployment_instruction.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Here is the asciinema I made. Full video of my console for installing SUSI AI Chat:
```
Comment (anshumanv) [inline docs/deployment_instruction.txt]: This can just go like - "Follow this video to view a sample console output during installation."

Comment (atiabjobayer) [reply docs/deployment_instruction.txt]: What should I do now? Should I edit it?

Comment (0x48piraj) [reply docs/deployment_instruction.txt]: It has already a deployment instruction :wink:

Comment (ms10398) [reply docs/deployment_instruction.txt]: You can change the text as @anshumanv said and rest it looks good

Comment (0x48piraj) [reply docs/deployment_instruction.txt]: One PR is already pending !

Comment (ms10398) [reply docs/deployment_instruction.txt]: Your PR had text instructions @0x48piraj his PR adds a ascinemma both PR together makes a great documentation

Comment (0x48piraj) [reply docs/deployment_instruction.txt]: Oh Great!

Comment (0x48piraj) [reply docs/deployment_instruction.txt]: `.txt` confused me :-1:

Comment (ms10398) [reply docs/deployment_instruction.txt]: But i think file names of both of your files is different so after both PR merges I will merge both files to make deploy.md

Comment (0x48piraj) [reply docs/deployment_instruction.txt]: Yep! I was also thinking of the same :wink:

**Comment by 0x48piraj** [comment]
@ParthS007 Can you please see https://github.com/fossasia/chat.susi.ai/issues/926

**Comment by ParthS007** [comment]
@0x48piraj  I am asking to provide the link in README.md so that it is more accessible.

**Comment by mariobehling** [comment]
Closing this due to inactivity. I will add the link myself.

**Pull request closed without merging**